qualtran/drawing/musical_score.py

```diff
@@ -476,7 +477,7 @@ def _soq_to_symb(soq: Soquet) -> WireSymbol:
 
     # Use text (with no box) for dangling register identifiers.
     if isinstance(soq.binst, DanglingT):
-        return Text(soq.pretty())
+        return Text(soq.pretty() + f'[{soq.reg.dtype!s}]', fontsize=8)
```

Comment:
is !s necessary? isn't that the default?

Reply:
you are right, it's not needed. Removed.